src/components/Sing/ScoreSequencer.vue

```diff
@@ -627,36 +631,64 @@ export default defineComponent({
       previewRequestId = requestAnimationFrame(preview);
     };
 
-    const onNoteBodyMouseDown = (event: MouseEvent, note: Note) => {
-      startPreview(event, "MOVE", note);
-      mouseDownNoteId = note.id;
+    const onNoteBarMouseDown = (event: MouseEvent, note: Note) => {
+      if (event.target !== event.currentTarget) {
+        return;
+      }
```

Comment:
この処理、関数化して意図のわかりやすい名称にしたいですね！
とりあえずTODOコメント書いておくとかどうでしょう。
```suggestion
      // 直接イベントが来ていない場合は無視する
      // TODO: 関数化
      if (event.target !== event.currentTarget) {
        return;
      }
```
（ほんとはstopPropagationが使えない理由も書きたいですが。。。）